src/pages/patientView/copyNumberAlterations/column/AnnotationColumnFormatter.tsx

```diff
@@ -32,13 +41,50 @@ export default class AnnotationColumnFormatter
             value = {
                 myCancerGenomeLinks: [],
                 isHotspot: false,
-                is3dHotspot: false
+                is3dHotspot: false,
+                hasCivicVariants: true
             };
         }
 
         return value;
     }
 
+   /**
+     * Returns an ICivicEntry if the civicGenes and civicVariants have information about the gene and the mutation (variant) specified. Otherwise it returns 
+     * an empty object.
+     */
+    public static getCivicEntry(copyNumberData:DiscreteCopyNumberData[], civicGenes:ICivicGene, civicVariants:ICivicVariant): ICivicEntry | null
```

Comment:
See my other comment in the `mutationTable/column/AnnotationColumnFormatter`related to duplication.